apps-devstg/us-east-1/databases-aurora/rds-export-to-s3/main.tf

```diff
@@ -11,20 +11,21 @@ locals {
 # RDS Export To S3
 # -----------------------------------------------------------------------------
 module "rds_export_to_s3" {
-  source = "github.com/binbashar/terraform-aws-rds-export-to-s3.git?ref=v0.3.0"
+  source = "github.com/binbashar/terraform-aws-rds-export-to-s3.git?ref=v0.4.0"
 
   # Set a prefix for naming resources
   prefix = "aurora-mysql"
 
+  create_customer_kms_key = true
+
   # The database name whose RDS snapshots will be exported to S3
-  database_name = data.terraform_remote_state.databases-aurora.outputs.this_rds_cluster_id
+  database_names = data.terraform_remote_state.databases-aurora.outputs.cluster_database_name
 
   # The RDS snapshots events that should be included: RDS Aurora (RDS-EVENT-0169) or RDS non-Aurora (RDS-EVENT-0091)
-  rds_event_id = "RDS-EVENT-0169"
+  rds_event_ids = "RDS-EVENT-0169, RDS-EVENT-0075"
```

Comment:
Since it appears to be that manual snapshots are also supported now, we would need to update the module's doc here: https://github.com/binbashar/terraform-aws-rds-export-to-s3/tree/master#input_rds_event_ids

Reply:
I made this small PR to solve this one: 
binbashar/terraform-aws-rds-export-to-s3/pull/11